Mage.Sets/src/mage/cards/t/TerisiaresDevastation.java

```diff
@@ -0,0 +1,44 @@
+package mage.cards.t;
+
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.dynamicvalue.common.ManacostVariableValue;
+import mage.abilities.dynamicvalue.common.PermanentsOnBattlefieldCount;
+import mage.abilities.effects.common.CreateTokenEffect;
+import mage.abilities.effects.common.LoseLifeSourceControllerEffect;
+import mage.abilities.effects.common.continuous.BoostAllEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.filter.common.FilterControlledArtifactPermanent;
+import mage.game.permanent.token.PowerstoneToken;
+
+import java.util.UUID;
+
+/**
+ * @author PurpleCrowbar
+ */
+public final class TerisiaresDevastation extends CardImpl {
+
+    private static final DynamicValue xValue = new PermanentsOnBattlefieldCount(new FilterControlledArtifactPermanent(), -1);
+
+    public TerisiaresDevastation(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.SORCERY}, "{X}{2}{B}{B}");
+
+        // You lose X life and create X tapped Powerstone tokens. Then all creatures get -1/-1 until end of turn for each artifact you control.
+        this.getSpellAbility().addEffect(new LoseLifeSourceControllerEffect(ManacostVariableValue.REGULAR));
+        this.getSpellAbility().addEffect(new CreateTokenEffect(new PowerstoneToken(), ManacostVariableValue.REGULAR, true, false).concatBy("and"));
+        this.getSpellAbility().addEffect(new BoostAllEffect(
+                xValue, xValue, Duration.EndOfTurn)
```

Comment:
These should be -xValue, -xValue

Reply:
The multiplier on line 23 inverts the sign by multiplying by -1, so xValue is a negative number

Reply:
You're right. I missed that.